style.css

```diff
@@ -0,0 +1,718 @@
+  margin: 0 3%;
+  background-image:
+    linear-gradient(
+      rgba(228, 43, 23, 0.5),
+      rgba(221, 43, 23, 0.5)
+    ),
+    url(./images/fruit-salad.jpg);
+  padding: 10% 20%;
+  background-size: cover;
+  background-repeat: no-repeat;
+  margin-bottom: 5%;
+}
+
+.feature-program-name2 {
+  margin: 0 3%;
+  background-image:
+    linear-gradient(
+      rgba(228, 43, 23, 0.5),
+      rgba(221, 43, 23, 0.5)
+    ),
+    url(./images/nutrtion\ seminar.png);
+  padding: 10% 20%;
+  background-size: cover;
+  background-repeat: no-repeat;
+  margin-bottom: 5%;
+}
+
+.feature-program-heading {
+  font-size: 20px;
+  font-weight: 900;
+  color: #f0f2f5;
+  text-align: center;
+  font-family: 'lato', sans-serif;
+}
+
+.feature-program-description {
+  font-weight: 900;
+  color: #f0f2f5;
+  text-align: center;
+}
+
+.project-rights-about {
+  background-color: #272a31;
+  display: flex;
+  align-items: center;
+  padding: 4%;
+}
+
+.footer-content {
+  color: #d3d3d3;
+}
+
+.footer-logo-about {
+  width: 30%;
+  height: 50%;
+  margin-right: 10%;
+}
+
+.footer-text {
+  font-family: "lato", sans-serif;
+  color: #272a31;
+  font-weight: 400;
+  margin-top: 20px;
+  font-size: 1.2rem;
+}
+
+@media only screen and (min-width: 769px) {
+  .featured-program-item-desktop {
+    background-color: rgba(88, 89, 90, 0.8);
+    display: flex;
+    justify-content: center;
+    flex-direction: column;
+    text-align: center;
+    margin-right: 0.5%;
+    width: 15%;
+    padding: 1%;
+  }
+
+  .nav-button-link {
+    border: #c21b09 solid 4px;
+    padding: 2%;
+  }
+
+  .nav-button {
+    color: #c21b09;
+  }
+
+  .mobile-hide {
+    display: block;
+  }
+
+  .mobile-works {
+    display: none;
+  }
+
+  .social-media-section {
+    background-color: #272a31;
+    align-items: center;
+  }
+
+  .social-media {
+    list-style: none;
+    align-items: center;
+    text-align: center;
+  }
+
+  .social-media-item {
+    text-decoration: none;
+    color: #d3d3d3;
+  }
+
+  .social-media-row {
+    align-items: center;
+    text-align: center;
+    padding-top: 0.8%;
+  }
+
+  .nav {
+    align-items: center;
+    text-align: center;
+    flex: 2;
+  }
+
+  .nav-list {
+    display: flex;
+  }
+
+  .nav-item {
+    list-style: none;
+    text-decoration: none;
+    color: black;
+  }
+
+  .nav-item:hover {
+    color: #c21b09;
+    text-decoration: none;
+  }
+
+  .header-logo {
+    width: 50%;
+    height: 100%;
+  }
+
+  .header-logo-link {
+    flex: 1;
+  }
+
+  .introduction {
+    padding-top: 10%;
+  }
+
+  .introduction-title {
+    font-size: 60px;
+    width: 70%;
+  }
+
+  .introduction-description {
+    width: 60%;
+  }
+
+  .introduction-offers {
+    font-size: 30px;
+  }
+
+  .introduction-time {
+    font-size: 25px;
+  }
+
+  .container-1 {
+    height: 480px;
+  }
+
+  .program-item-holder {
+    display: flex;
+    padding: 3% 5%;
+    margin-bottom: 40px;
+    gap: 1%;
+    margin-top: 0;
+    margin-left: 5%;
+  }
```

Comment:
- [ ] Let's make sure the main program cards are centered on the page, currently they doesn't not look centered please see :point_down: 
![image](https://user-images.githubusercontent.com/97521557/217485582-97a43f6f-4f70-49b5-b4a3-03f6d195d31b.png)

It is an easy fix, just use `justify-content: center` and it will make sure the content is horizontally centered